stripe/src/main/java/com/stripe/android/util/StripeTextUtils.java

```diff
@@ -100,4 +101,28 @@ public static String asCardBrand(@Nullable String possibleCardType) {
             return Card.UNKNOWN;
         }
     }
+
+    /**
+     * Converts an unchecked String value to a {@link FundingType} or {@code null}.
+     *
+     * @param possibleFundingType a String that might match a {@link FundingType} or be empty
+     * @return {@code null} if the input is blank, else the appropriate {@link FundingType}
+     */
+    @Nullable
+    @FundingType
+    public static String asFundingType(@Nullable String possibleFundingType) {
```

Comment:
Just out of curiosity, who's meant to be the main user of this method? Is it meant to be a utility to help users integrate more easily? If not, it might be just as good to only allow a strict subset of the values in `FundingType`.